src/content/reference/manifest.mdx

```diff
@@ -509,6 +509,31 @@ metadata:
     custom.label/dev: "true"
 ```
 
+## mode (string, optional)
+
+Defines how `okteto up` will setup the development container and behave within the cluster.
+
+Currently two options are available:
+
+- `sync`: (default) Sync & run your code in a remote development container.
+- `hybrid`: Run your code on your local computer instead of in a remote development container, while rest of your service components runs in your remote cluster.
+
+|                   | `sync` (default) | `hybrid` |
```

Comment:
I think we have the same table in `cloud/okteto-cli`. Should we link that table here instead of repeating it?

Reply:
My vote would be to repeat it because I think it's important in both sections and is succinct.

If the content was more verbose then I'd say link to it, but since this is fairly concise I think it's worth saving the user a trip to another doc page so they can stay in the same context.

Happy to hear contrary opinion though 😄